src/main/java/org/opentripplanner/apis/transmodel/model/EnumTypes.java

```diff
@@ -133,13 +134,40 @@ public class EnumTypes {
 
   public static final GraphQLEnumType INTERCHANGE_WEIGHTING = GraphQLEnumType
     .newEnum()
+    .description("Deprecated. Use STOP_INTERCHANGE_PRIORITY")
     .name("InterchangeWeighting")
     .value("preferredInterchange", 2, "Highest priority interchange.")
     .value("recommendedInterchange", 1, "Second highest priority interchange.")
     .value("interchangeAllowed", 0, "Third highest priority interchange.")
     .value("noInterchange", -1, "Interchange not allowed.")
     .build();
 
+  public static final GraphQLEnumType STOP_INTERCHANGE_PRIORITY = GraphQLEnumType
+    .newEnum()
+    .name("StopInterchangePriority")
+    .value(
+      "preferred",
+      StopTransferPriority.PREFERRED,
+      "Preferred place to transfer, strongly recommended. NeTEx equivalent is PREFERRED_INTERCHANGE."
```

Comment:
Woudl it be possible to use `@DocumentedEnum` here?

Reply:
I'm not sure what you are referring to?

Reply:
Rather than using Javadoc which is then copied to the GraphQL schema, you can annotate the enum and have only a single place for the documentation, like this: https://github.com/opentripplanner/OpenTripPlanner/blob/46a347c5339e79a0e7d3123f8118bf8bbe64db42/src/main/java/org/opentripplanner/transit/model/basic/TransitMode.java#L62-L83

It's particularly useful if you have the information in both the HTML documentation as well as the schema, so it's usefulness isn't quite so high here.